sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_SLM.cpp

```diff
@@ -0,0 +1,26 @@
+//==--- joint_matrix_bf16_fill_k_cache_SLM.cpp  - DPC++ joint_matrix--------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: aspect-ext_intel_matrix, gpu
+
+// RUN: %{build} -o %t_gpu_vnni.out -ffp-model=precise -DSLM -DVNNI
+// RUN: %{run} %t_gpu_vnni.out
+
+// RUN: %{build} -o %t_gpu.out -ffp-model=precise -DSLM
+// RUN: %{run} %t_gpu.out
+
+// RUN: %{build} -o %t_gpu_ocl_vnni.out -ffp-model=precise -DSLM -DOCL -DVNNI 
```

Comment:
In my opinion, we don't need to keep both SYCL and OCL variants of SLM load/store in Joint Matrix tests.
The goal is to test Joint Matrix, not SLM load/stores, so we need to select the variant, which we think is the best at this point and use only it. If later better solution of SLM load/store appears, we can update test implementation.